python/sglang/srt/two_batch_overlap.py

```diff
@@ -317,6 +364,9 @@ def filter_batch(
             old_value = getattr(batch, key)
             if old_value is None:
                 continue
+            elif batch.forward_mode.is_target_verify() and key.startswith("extend_"):
```

Comment:
nit: a bit worried whether this is too dynamic. I would personally suggest listing all extend_blahblah manually to the existing long list, probably in a separate section with a comment